### ISSUE TITLE
Some Symphony Markdown Showing on Front-end

### ISSUE THREAD

**Issue by davidfrost** [atheycreek/atheycreek#52]
On the events page ( http://atheycreek.com/events/ ), the description for the "SAVE THE DATE - CAMP BRADLEY" Jr. High event is showing some of the markdown code. 

The actual page for the event ( http://atheycreek.com/events/16330/save-the-date-camp-bradley/ ) does NOT show any of the markdown. 

Symphony entry for this event - http://atheycreek.com/admin/publish/events/edit/16330/

![screenshot 2015-06-14 09 13 23](https://cloud.githubusercontent.com/assets/1236878/8149370/9f8a8b4a-1275-11e5-83b4-424b4968bf3e.png)

**Comment by jdsimcoe** [comment]
That's normal and expected... there's no way around that really.

**Comment by davidfrost** [comment]
Wow. Really?


On Jun 14, 2015, at 9:32 AM, Jonathan Simcoe <notifications@github.com<mailto:notifications@github.com>> wrote:

That's normal and expected... there's no way around that really.

**Comment by jdsimcoe** [comment]
Yeah there's really no easy way around that I know of. Most of the time it
doesn't show because the main description is longer.
